internal/livelint/run_checks.go

```diff
@@ -35,12 +35,18 @@ func (n *Livelint) RunChecks(namespace, deploymentName string, isVerbose bool) e
 		return fmt.Errorf("error getting Deployment Pods: %w", err)
 	}
 
-	// Are you hitting the ResourceQuota limits?
-	result := n.checkAreResourceQuotasHit(namespace, deploymentName)
+	// Is the number of running pods correct ?
+	result := n.CheckIsNumberOfPodsMatching(namespace, deploymentName)
 	statusMsg.AddCheckResult(result)
 	n.ui.Send(statusMsg)
 	if result.HasFailed {
-		return nil
+		// Are you hitting the ResourceQuota limits?
+		result = n.checkAreResourceQuotasHit(namespace, deploymentName)
```

Comment:
This doesn't quite follow the logic as described in the issue. The quota should only be checked if there are too few pods. However, here the check will also be run if there are too many pods.

Although, with the current architecture, it may not be possible to make that distinction in a sensible fashion.

Reply:
Discussed with This, we still want to run checks if there are too many pods. I added a warning option.